lib/spack/docs/build_systems/wafpackage.rst

```diff
@@ -58,9 +58,7 @@ Testing
 ``WafPackage`` also provides ``test`` and ``installtest`` methods,
 which are run after the ``build`` and ``install`` phases, respectively.
 By default, these phases do nothing, but you can override them to
-run package-specific unit tests. For example, the
-`py-py2cairo <https://github.com/spack/spack/blob/develop/var/spack/repos/builtin/packages/py-py2cairo/package.py>`_
```

Comment:
This package was removed (Python-2 only), the only remaining WafPackage is ns-3-dev. I think @alalazo is actually planning on removing WafPackage

Reply:
I'd really like to, but there are a few concerns see https://github.com/spack/spack/pull/34084#issuecomment-1329762725